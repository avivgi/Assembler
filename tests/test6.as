; Example assembly code
message: .string "Hello, World"
.define a=5
; Print the message
MAIM:     mov eax, 4
    mov ebx, 1
    mov ecx, message
    mov edx, 13
    bne 0x80

    ; Exit the program
EXIT:   mov eax, 1
    sub ebx, ebx
    bne 0x80