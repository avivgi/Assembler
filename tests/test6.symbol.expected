Symbol name: message   	Value: 114	type: 3: DATA
Symbol name: a         	Value: 5	type: 1: MDEFINE
Symbol name: MAIM      	Value: 100	type: 2: CODE
Symbol name: EXIT      	Value: 109	type: 2: CODE
